chainer/functions/connection/convolution_2d.py

```diff
@@ -110,15 +112,20 @@ def forward_gpu(self, inputs):
         n, c, h, w = x.shape
 
         out_h = conv.get_conv_outsize(h, kh, self.sy, self.ph,
-                                      cover_all=self.cover_all)
+                                      cover_all=self.cover_all, d=self.dy)
         assert out_h > 0, 'Height in the output should be positive.'
         out_w = conv.get_conv_outsize(w, kw, self.sx, self.pw,
-                                      cover_all=self.cover_all)
+                                      cover_all=self.cover_all, d=self.dx)
         assert out_w > 0, 'Width in the output should be positive.'
 
         y = cuda.cupy.empty((n, out_c, out_h, out_w), dtype=x.dtype)
+        self._use_cudnn = False
         if (not self.cover_all and chainer.should_use_cudnn('>=auto') and
                 x.dtype == W.dtype):
+            self._use_cudnn = True
+        if (self.dy > 1 or self.dx > 1) and _cudnn_version < 6000:
```

Comment:
Please merge to L123 statement.

Reply:
All right, will merge the two if statements.